Error while checking build/O2/fullCI_slc9 for 00c55fd6bee6348a0ba3db473d8d97b066353f26 at 2025-09-24 12:23:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/14687-slc9_x86-64/0/Framework/Core/src/HTTPParser.cxx:226:27: error: 'protocol' was not declared in this scope
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14687/00c55fd6bee6348a0ba3db473d8d97b066353f26/build_O2_fullCI_slc9/pretty.html).

--- a/Framework/Core/src/HTTPParser.cxx
+++ b/Framework/Core/src/HTTPParser.cxx
@@ -214,7 +214,7 @@ std::string HTTPParserHelpers::calculateAccept(const char* nonce)
   return fmt::format("{}", base);
 }
 
-std::string encode_websocket_handshake_reply(char const* nonce)
+std::string encode_websocket_handshake_reply(char const* nonce, const char* protocol)
 {
   constexpr auto res =
     "HTTP/1.1 101 Switching Protocols\r\n"

